client/web/src/enterprise/insights/pages/insights/insight/components/actions/CodeInsightIndependentPageActions.tsx

```diff
@@ -46,6 +46,12 @@ export const CodeInsightIndependentPageActions: FunctionComponent<Props> = props
 
     return (
         <div className={styles.container}>
+            {!isLangStatsInsight(insight) && (
+                <Button as={Link} to={`/api/insights/export/${insight.id}`} variant="secondary">
+                    Export CVS data
```

Comment:
```suggestion
                    Export data as CSV
```
I think that's a more standard way of saying it